odoo/fields.py

```diff
@@ -1661,19 +1569,16 @@ def context_today(record, timestamp=None):
 
     @staticmethod
     def to_date(value):
-        """
-        Attempt to convert ``value`` to a :class:`date` object.
+        r"""Attempt to convert ``value`` to a :class:`date` object.
```

Comment:
`r`?

Reply:
RCO request, remove the raw and update the docstring to escape the \ symbol.